include/wx/arrstr.h

```diff
@@ -16,6 +16,9 @@
 #include "wx/dynarray.h"
 
 #include <vector>
```

Comment:
Small inconsistency here: I didn't want to include `<vector>` between the `beforestd` and `afterstd` guards. Is that expected?

Reply:
It doesn't really matter in practice because there are no known problems with including `<vector>` outside of them (and very few problems in standard headers remain at all, so we probably could include `<initializer_list>` outside of them too if we really wanted).

Reply:
Removed the std guards